src/dataprotection/README.md

```diff
@@ -0,0 +1,201 @@
+# Azure CLI dataprotection Extension #
+This is the extension for dataprotection
+
+### How to use ###
+Install this extension using the below CLI command
+```
+az extension add --name dataprotection
+```
+
+### Included Features ###
+#### dataprotection backup-vault ####
+##### Create #####
+```
+az dataprotection backup-vault create --type "None" --location "WestUS" \
+    --storage-settings type="LocallyRedundant" datastore-type="VaultStore" --tags key1="val1" \
+    --resource-group "SampleResourceGroup" --vault-name "swaggerExample" 
+```
+##### Create #####
+```
+az dataprotection backup-vault create --type "systemAssigned" --location "WestUS" \
+    --storage-settings type="LocallyRedundant" datastore-type="VaultStore" --tags key1="val1" \
+    --resource-group "SampleResourceGroup" --vault-name "swaggerExample" 
+```
+##### Show #####
+```
+az dataprotection backup-vault show --resource-group "SampleResourceGroup" --vault-name "swaggerExample"
+```
+##### Show #####
+```
+az dataprotection backup-vault show --resource-group "SampleResourceGroup" --vault-name "swaggerExample"
+```
+##### Update #####
+```
+az dataprotection backup-vault update --tags newKey="newVal" --resource-group "SampleResourceGroup" \
+    --vault-name "swaggerExample" 
+```
+##### Show-in-resource-group #####
+```
+az dataprotection backup-vault show-in-resource-group --resource-group "SampleResourceGroup"
+```
+##### Show-in-subscription #####
+```
+az dataprotection backup-vault show-in-subscription
+```
+##### Delete #####
+```
+az dataprotection backup-vault delete --resource-group "SampleResourceGroup" --vault-name "swaggerExample"
+```
+#### dataprotection operation-status ####
+##### Show #####
+```
+az dataprotection operation-status show \
+    --operation-id "MjkxOTMyODMtYTE3My00YzJjLTg5NjctN2E4MDIxNDA3NjA2OzdjNGE2ZWRjLWJjMmItNDRkYi1hYzMzLWY1YzEwNzk5Y2EyOA==" \
+    --location "WestUS" 
+```
+#### dataprotection backup-vault-operation-result ####
+##### Show #####
+```
+az dataprotection backup-vault-operation-result show \
+    --operation-id "YWUzNDFkMzQtZmM5OS00MmUyLWEzNDMtZGJkMDIxZjlmZjgzOzdmYzBiMzhmLTc2NmItNDM5NS05OWQ1LTVmOGEzNzg4MWQzNA==" \
+    --resource-group "SampleResourceGroup" --vault-name "swaggerExample" 
+```
```

Comment:
The command `az dataprotection backup-vault-operation-result show` is strange. I think it should be `az dataprotection backup-vault show-operation-result`

Reply:
updated readme file